test/ntriples-star/ntriples-star-bad-syntax-1.nt

```diff
@@ -0,0 +1 @@
+<http://example/a> << <http://example/s> <http://example/p>  <http://example/o> >>  <http://example/z> .
```

Comment:
These files should be relocated to [`test/data/`](https://github.com/RDFLib/rdflib/tree/main/test/data) and [`test/data/fetcher.py`](https://github.com/RDFLib/rdflib/blob/main/test/data/fetcher.py) should be updated to fetch it.

For more info see [`test/data/README.md`](https://github.com/RDFLib/rdflib/blob/main/test/data/README.md)

Reply:
I'm happy to move it if you are fine with it.